### ISSUE TITLE
No response after injection

### ISSUE THREAD

**Issue by djkcyl** [danielkrupinski/Osiris#1900]
Csgo nothing happens after I inject using the injector

**Comment by king1919djdj** [comment]
which injector are u using ?

**Comment by king1919djdj** [comment]
btw u should use this one #1862

**Comment by djkcyl** [comment]
It also cannot be used

**Comment by king1919djdj** [comment]
why it cant be used ?
i used it an hour ago

**Comment by djkcyl** [comment]
Nothing happens after I inject
Let me try again

**Comment by king1919djdj** [comment]
download the latest source
build it as told on the injector page
rename ur osiris.dll to cheat.dll
place it in the folder that contains the release(built from source)
run the csgoinjection.exe as admin

**Comment by djkcyl** [comment]
I can use it, thank you

**Comment by ousamaAdmin** [comment]
Doesn't work, it says ...
"cannot find processIDinjection failed!"
